00 Base/src/components/myComponent/myComponent.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  fetchfruits,
+  deleteFruit,
+  addFruit,
+} from './../../redux/actions/fruitsActions';
+import './styles.scss';
+
+import Button from '@material-ui/core/Button';
+import Chip from '@material-ui/core/Chip';
+
+import { MyInputComponent } from '../myInputComponent/myInputComponent';
+export interface Props {
+  nameFromProps: string;
+}
+
+export const MyComponent: React.FunctionComponent<Props> = (props) => {
+  const { nameFromProps } = props;
+  const dispatch = useDispatch();
+  const fruitList = useSelector((state) => state.fruits.fruitList);
+  console.log('fruitList', fruitList);
+
+  return (
+    <>
+      <div className="container">
+        <h1>Hello {nameFromProps}!</h1>
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={() => dispatch(fetchfruits())}
+        >
+          Get some fruits!
+        </Button>
+        <div className="chip-container">
+          {!!fruitList
+            ? fruitList.map((el: string) => (
```

Comment:
I can see this is not painting anything
![image](https://user-images.githubusercontent.com/14187188/78788012-7b1ba800-79ab-11ea-878f-946a46f389a3.png)